CESC-submission/paper.tex

```diff
@@ -208,64 +208,61 @@ \section{Impact of Financial Incentives on Service Delivery}
+\paragraph{Confirm availability solution.} Here we use a detect-and-punish approach embedded within an optimistic service monitoring process. This means that it is only invoked when Bob complains that an Ursula is unresponsive. In detail, for each round\footnote{A round is the time needed to mine a block or several blocks on the underlying blockchain.} a set of Ursulas is selected at random to act as gateways (or service watchers).\footnote{One approach to do that in a distributed way is to feed a random beacon (like block hashes in the random oracle model) to an iterative hashing process. Then map each hash to an Ursula index as found in the list of working Ursulas recorded on the blockchain within the StakeEscrow smart contract of NuCypher. This process is repeated until a set of distinct Ursulas is selected. Note that anyone can perform this process and verify its outcome.} Bob proceeds as usual by contacting the list of working Ursulas received from Alice for the re-encryption service. If a working Ursula does not reply, Bob complains to a gateway Ursula and hands her the request. This gateway mediates the service process in order to check if Bob's claim is valid. In other words, she challenges the working Ursula by forwarding the request again and waiting for an answer (which must arrive within a specified, universal time window). If one is received, the gateway forwards it to Bob and closes the complaint.
 
 
-On the other hand, meaning no response is received, the gateway notifies the rest of gateways selected for that round to repeat the same process. If no response is received by any of them, the quorum signs a proof-of-unavailability (e.g., by using collective signing~\cite{syta2016keeping}) against the working Ursula stating that she is unavailable and submit this proof to the NuCypher staking contract. At that time, a part of the working Ursula stake is slashed (with a value derived based on the additional utility gain an Ursula collects by faking to be online) as a punishment. This punishment makes cheating unprofitable, and hence, unappealing to rational working Ursulas. A detailed technical version of this solution is available in~\cite{confirm-activity-draft}.
+On the other hand, meaning no response is received, the gateway notifies the rest of gateways selected for that round to repeat the same process. If no response is received by any of them, the quorum signs a proof-of-unavailability (e.g., by using collective signing~\cite{syta2016keeping}) against the working Ursula stating that she is unavailable and submit this proof to the NuCypher staking contract. At that time, a portion of the Ursula's stake is slashed (with a minimum value derived based on the additional utility gain an Ursula collects by faking to be online) as a punishment. This punishment makes cheating unprofitable, and hence, unappealing to rational Ursulas. A detailed technical version of this solution is available in~\cite{confirm-activity-draft}.
```

Comment:
At that time, a part of the working Ursula stake -> At that time, a portion of the Ursula's stake